rake_nltk/rake.py

```diff
@@ -166,20 +166,12 @@ def _generate_phrases(self, sentences):
            @return: List of List of strings where each sublist is a collection
```

Comment:
Here we should maybe even say `@return: Set of string tuples...` because the return value is now a set instead of a list.